client/src/main/scala/org/http4s/client/Client.scala

```diff
@@ -192,4 +200,50 @@ final case class Client(open: Service[Request, DisposableResponse], shutdown: Ta
     shutdown.run
 }
 
+<<<<<<< HEAD
+object Client {
+  /** Creates a mock client from the specified service.  Useful for generating
+    * pre-determined responses for requests in testing.
+    * 
+    * @param service the service to respond to requests to this client
+    */
+  def mock(service: HttpService): Client = {
```

Comment:
Can we make http4s-core's tests depend on http4s-test which depends on http4s-core main?  Do we have to sacrifice having tests in their own modules to use our own test library, if we create one?